types/sonic-boom/index.d.ts

```diff
@@ -7,9 +7,7 @@
 
 import { EventEmitter } from 'events';
 
-export = SonicBoom;
```

Comment:
`SonicBoom` is a CommonJS module (as you can see [here](https://github.com/mcollina/sonic-boom/blob/master/index.js#L219)) and should be export as before `export = SonicBoom`